internal/metrics/metricsManager.go

```diff
@@ -370,7 +370,7 @@ func (m *MetricsManager) ClusterHostInstallationCount(clusterID strfmt.UUID, ema
 func (m *MetricsManager) ClusterInstallationFinished(ctx context.Context, result, clusterVersion string, clusterID strfmt.UUID, emailDomain string, installationStartedTime strfmt.DateTime) {
 	duration := time.Since(time.Time(installationStartedTime)).Seconds()
 	m.handler.AddMetricsEvent(ctx, clusterID, nil, models.EventSeverityInfo, "cluster.installation.results", time.Now(),
-		"duration", duration)
+		"duration", duration, "result", result)
```

Comment:
prefix it with something meaningful because it will get the same mapping as other results. Something like cluster_install_result